# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/if_arp_h.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2004 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

needed by iproute2

--- ./include/net/if_arp.h.orig	Sat Jan 11 13:09:07 2003
+++ ./include/net/if_arp.h	Sat Jan 11 13:09:19 2003
@@ -18,6 +18,7 @@
 #define ARPHRD_DLCI	15		/* Frame Relay DLCI		*/
 #define ARPHRD_ATM	19		/* ATM 				*/
 #define ARPHRD_METRICOM	23		/* Metricom STRIP (new IANA id)	*/
+#define ARPHRD_IEEE1394 24              /* IEEE 1394 IPv4 - RFC 2734    */
 
 /* Dummy types for non ARP hardware */
 #define ARPHRD_SLIP	256
